### ISSUE TITLE
Test failure in Microsoft.CodeAnalysis.LanguageServer.UnitTests.Miscellaneous.LspMiscellaneousFilesWorkspaceTests.TestLooseFile_MovedToRegisteredWorkspace

### ISSUE THREAD

**Issue by allisonchou** [dotnet/roslyn#57555]
Link to build: https://runfo.azurewebsites.net/view/build/?number=1450058

Example output from Helix log:

```
Console log: 'Microsoft.CodeAnalysis.LanguageServer.Protocol.UnitTests.dll' from job 4d2ed691-9413-4b85-9b1c-5ac646029953 workitem bd7b5f19-98ce-443b-a7a7-5b58bed5099d (windows.10.amd64.server19h1.es.open.svc) executed on machine a000JNL

C:\h\w\A62208C0\w\B8B609E9\e>set PATH=C:\h\w\A62208C0\p\dotnet-cli;C:\python3.9.1\Scripts\;C:\python3.9.1\;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Users\runner\AppData\Local\Microsoft\WindowsApps 

C:\h\w\A62208C0\w\B8B609E9\e>set DOTNET_ROOT=C:\h\w\A62208C0\p\dotnet-cli 

C:\h\w\A62208C0\w\B8B609E9\e>set DOTNET_CLI_TELEMETRY_OPTOUT=1 

C:\h\w\A62208C0\w\B8B609E9\e>set DOTNET_CLI_HOME=C:\h\w\A62208C0\w\B8B609E9\e\.dotnet 

C:\h\w\A62208C0\w\B8B609E9\e>set NUGET_PACKAGES=C:\h\w\A62208C0\w\B8B609E9\e\.nuget 

C:\h\w\A62208C0\w\B8B609E9\e>dir
 El volumen de la unidad C no tiene etiqueta.
 El n�mero de serie del volumen es: A41D-76BB

 Directorio de C:\h\w\A62208C0\w\B8B609E9\e

02/11/2021  06:47    <DIR>          .
02/11/2021  06:47    <DIR>          ..
02/11/2021  06:47             9.625 Microsoft.CodeAnalysis.EditorFeatures.Test.Utilities.dll.config
02/11/2021  06:47             4.501 Microsoft.CodeAnalysis.EditorFeatures.Wpf.dll.config
02/11/2021  06:47           453.120 Microsoft.CodeAnalysis.LanguageServer.Protocol.UnitTests.dll
02/11/2021  06:47             9.987 Microsoft.CodeAnalysis.LanguageServer.Protocol.UnitTests.dll.config
02/11/2021  06:47             2.143 Microsoft.CodeAnalysis.Test.Utilities.dll.config
02/11/2021  06:47             2.131 Microsoft.CodeAnalysis.Workspaces.MSBuild.dll.config
02/11/2021  06:47             8.925 Microsoft.CodeAnalysis.Workspaces.Test.Utilities.dll.config
02/11/2021  06:47    <DIR>          ref
02/11/2021  06:47           208.893 rehydrate.cmd
02/11/2021  06:47    <DIR>          runtimes
02/11/2021  06:47                99 xunit.runner.json
               9 archivos        699.424 bytes
               4 dirs  119.866.667.008 bytes libres

C:\h\w\A62208C0\w\B8B609E9\e>call .\rehydrate.cmd 

C:\h\w\A62208C0\w\B8B609E9\e>dir
 El volumen de la unidad C no tiene etiqueta.
 El n�mero de serie del volumen es: A41D-76BB

 Directorio de C:\h\w\A62208C0\w\B8B609E9\e

02/11/2021  06:47    <DIR>          .
02/11/2021  06:47    <DIR>          ..
02/11/2021  06:34         4.654.592 Basic.Reference.Assemblies.Net50.dll
02/11/2021  06:34         5.023.232 Basic.Reference.Assemblies.Net60.dll
02/11/2021  06:34         3.890.688 Basic.Reference.Assemblies.NetStandard20.dll
02/11/2021  06:34            17.408 BasicUndo.dll
02/11/2021  06:34           434.688 Castle.Core.dll
02/11/2021  06:47    <DIR>          cs
02/11/2021  06:47    <DIR>          de
02/11/2021  06:34            19.456 DiffPlex.dll
02/11/2021  06:47    <DIR>          en
02/11/2021  06:47    <DIR>          es
02/11/2021  06:47    <DIR>          fr
02/11/2021  06:34           212.008 Humanizer.dll
02/11/2021  06:34         1.837.568 ICSharpCode.Decompiler.dll
02/11/2021  06:47    <DIR>          it
02/11/2021  06:47    <DIR>          ja
02/11/2021  06:47    <DIR>          ko
02/11/2021  06:34             8.704 MessagePack.Annotations.dll
02/11/2021  06:34           296.960 MessagePack.dll
02/11/2021  06:34            20.872 Microsoft.Bcl.AsyncInterfaces.dll
02/11/2021  06:34           136.272 Microsoft.Build.Framework.dll
02/11/2021  06:34           301.944 Microsoft.CodeAnalysis.Analyzer.Testing.dll
02/11/2021  06:34           543.624 Microsoft.CodeAnalysis.AnalyzerUtilities.dll
02/11/2021  06:34            88.440 Microsoft.CodeAnalysis.CodeFix.Testing.dll
02/11/2021  06:34            59.768 Microsoft.CodeAnalysis.CodeRefactoring.Testing.dll
02/11/2021  06:34        13.840.384 Microsoft.CodeAnalysis.Compiler.Test.Resources.dll
02/11/2021  06:34            23.416 Microsoft.CodeAnalysis.CSharp.Analyzer.Testing.dll
02/11/2021  06:34            21.880 Microsoft.CodeAnalysis.CSharp.Analyzer.Testing.XUnit.dll
02/11/2021  06:34            23.928 Microsoft.CodeAnalysis.CSharp.CodeFix.Testing.dll
02/11/2021  06:34            22.392 Microsoft.CodeAnalysis.CSharp.CodeFix.Testing.XUnit.dll
02/11/2021  06:34            23.928 Microsoft.CodeAnalysis.CSharp.CodeRefactoring.Testing.dll
02/11/2021  06:34            22.928 Microsoft.CodeAnalysis.CSharp.CodeRefactoring.Testing.XUnit.dll
02/11/2021  06:34         7.019.520 Microsoft.CodeAnalysis.CSharp.dll
02/11/2021  06:34           160.768 Microsoft.CodeAnalysis.CSharp.EditorFeatures.dll
02/11/2021  06:34         1.693.184 Microsoft.CodeAnalysis.CSharp.Features.dll
02/11/2021  06:34            31.232 Microsoft.CodeAnalysis.CSharp.Scripting.dll
02/11/2021  06:34           943.104 Microsoft.CodeAnalysis.CSharp.Workspaces.dll
02/11/2021  06:34         3.032.576 Microsoft.CodeAnalysis.dll
02/11/2021  06:34         1.317.888 Microsoft.CodeAnalysis.EditorFeatures.dll
02/11/2021  06:34           611.328 Microsoft.CodeAnalysis.EditorFeatures.Test.Utilities.dll
02/11/2021  06:47             9.625 Microsoft.CodeAnalysis.EditorFeatures.Test.Utilities.dll.config
02/11/2021  06:34            56.320 Microsoft.CodeAnalysis.EditorFeatures.Text.dll
02/11/2021  06:34           376.320 Microsoft.CodeAnalysis.EditorFeatures.Wpf.dll
02/11/2021  06:47             4.501 Microsoft.CodeAnalysis.EditorFeatures.Wpf.dll.config
02/11/2021  06:34           145.320 Microsoft.CodeAnalysis.Elfie.dll
02/11/2021  06:34            80.384 Microsoft.CodeAnalysis.ExternalAccess.Razor.dll
02/11/2021  06:34         4.884.480 Microsoft.CodeAnalysis.Features.dll
02/11/2021  06:34           142.336 Microsoft.CodeAnalysis.InteractiveHost.dll
02/11/2021  06:34           205.312 Microsoft.CodeAnalysis.LanguageServer.Protocol.dll
02/11/2021  06:47           453.120 Microsoft.CodeAnalysis.LanguageServer.Protocol.UnitTests.dll
02/11/2021  06:47             9.987 Microsoft.CodeAnalysis.LanguageServer.Protocol.UnitTests.dll.config
02/11/2021  06:34           373.760 Microsoft.CodeAnalysis.Remote.ServiceHub.dll
02/11/2021  06:34           177.152 Microsoft.CodeAnalysis.Remote.Workspaces.dll
02/11/2021  06:34           155.648 Microsoft.CodeAnalysis.Scripting.dll
02/11/2021  06:34         2.484.224 Microsoft.CodeAnalysis.Test.Resources.Proprietary.dll
02/11/2021  06:34        25.154.560 Microsoft.CodeAnalysis.Test.Utilities.dll
02/11/2021  06:47             2.143 Microsoft.CodeAnalysis.Test.Utilities.dll.config
02/11/2021  06:34            29.560 Microsoft.CodeAnalysis.Testing.Verifiers.XUnit.dll
02/11/2021  06:34            23.416 Microsoft.CodeAnalysis.VisualBasic.Analyzer.Testing.dll
02/11/2021  06:34            21.880 Microsoft.CodeAnalysis.VisualBasic.Analyzer.Testing.XUnit.dll
02/11/2021  06:34            23.928 Microsoft.CodeAnalysis.VisualBasic.CodeFix.Testing.dll
02/11/2021  06:34            21.880 Microsoft.CodeAnalysis.VisualBasic.CodeFix.Testing.XUnit.dll
02/11/2021  06:34            24.456 Microsoft.CodeAnalysis.VisualBasic.CodeRefactoring.Testing.dll
02/11/2021  06:34            22.392 Microsoft.CodeAnalysis.VisualBasic.CodeRefactoring.Testing.XUnit.dll
02/11/2021  06:34         5.767.168 Microsoft.CodeAnalysis.VisualBasic.dll
02/11/2021  06:34           195.584 Microsoft.CodeAnalysis.VisualBasic.EditorFeatures.dll
02/11/2021  06:34         1.246.208 Microsoft.CodeAnalysis.VisualBasic.Features.dll
02/11/2021  06:34           905.216 Microsoft.CodeAnalysis.VisualBasic.Workspaces.dll
02/11/2021  06:34         3.668.480 Microsoft.CodeAnalysis.Workspaces.dll
02/11/2021  06:34           135.680 Microsoft.CodeAnalysis.Workspaces.MSBuild.dll
02/11/2021  06:47             2.131 Microsoft.CodeAnalysis.Workspaces.MSBuild.dll.config
02/11/2021  06:34           100.864 Microsoft.CodeAnalysis.Workspaces.Test.Utilities.dll
02/11/2021  06:47             8.925 Microsoft.CodeAnalysis.Workspaces.Test.Utilities.dll.config
02/11/2021  06:34            10.240 Microsoft.CodeAnalysis.XunitHook.dll
02/11/2021  06:34           170.288 Microsoft.Diagnostics.Tracing.EventSource.dll
02/11/2021  06:34           108.944 Microsoft.DiaSymReader.Converter.Xml.dll
02/11/2021  06:34            99.376 Microsoft.DiaSymReader.dll
02/11/2021  06:34            95.616 Microsoft.DiaSymReader.PortablePdb.dll
02/11/2021  06:34           117.640 Microsoft.Metadata.Visualizer.dll
02/11/2021  06:34           207.720 Microsoft.ServiceHub.Client.dll
02/11/2021  06:34           135.528 Microsoft.ServiceHub.Framework.dll
02/11/2021  06:34            44.408 Microsoft.ServiceHub.Resources.dll
02/11/2021  06:34            22.808 Microsoft.VisualStudio.CodeCoverage.Shim.dll
02/11/2021  06:34            19.304 Microsoft.VisualStudio.ComponentModelHost.dll
02/11/2021  06:34           372.088 Microsoft.VisualStudio.Composition.dll
02/11/2021  06:34            16.744 Microsoft.VisualStudio.Composition.NetFxAttributes.dll
02/11/2021  06:34           145.256 Microsoft.VisualStudio.CoreUtility.dll
02/11/2021  06:34            40.336 Microsoft.VisualStudio.Debugger.Contracts.dll
02/11/2021  06:34            53.608 Microsoft.VisualStudio.Editor.dll
02/11/2021  06:34           392.064 Microsoft.VisualStudio.GraphModel.dll
02/11/2021  06:34           475.000 Microsoft.VisualStudio.ImageCatalog.dll
02/11/2021  06:34           244.112 Microsoft.VisualStudio.Imaging.dll
02/11/2021  06:34            22.264 Microsoft.VisualStudio.Imaging.Interop.14.0.DesignTime.dll
02/11/2021  06:34           146.304 Microsoft.VisualStudio.InteractiveWindow.dll
02/11/2021  06:34         2.368.912 Microsoft.VisualStudio.Interop.dll
02/11/2021  06:34            18.872 Microsoft.VisualStudio.Language.CallHierarchy.dll
02/11/2021  06:34           188.280 Microsoft.VisualStudio.Language.dll
02/11/2021  06:34           151.928 Microsoft.VisualStudio.Language.Intellisense.dll
02/11/2021  06:34            30.056 Microsoft.VisualStudio.Language.NavigateTo.Interfaces.dll
02/11/2021  06:34            24.448 Microsoft.VisualStudio.Language.StandardClassification.dll
02/11/2021  06:34            33.144 Microsoft.VisualStudio.LanguageServer.Client.dll
02/11/2021  06:34           212.840 Microsoft.VisualStudio.LanguageServer.Protocol.dll
02/11/2021  06:34            35.192 Microsoft.VisualStudio.LanguageServer.Protocol.Extensions.dll
02/11/2021  06:34            73.592 Microsoft.VisualStudio.LanguageServer.Protocol.Internal.dll
02/11/2021  06:34         3.671.424 Microsoft.VisualStudio.Platform.VSEditor.dll
02/11/2021  06:34            20.240 Microsoft.VisualStudio.ProjectAggregator.dll
02/11/2021  06:34            45.952 Microsoft.VisualStudio.RemoteControl.dll
02/11/2021  06:34           105.848 Microsoft.VisualStudio.RpcContracts.dll
02/11/2021  06:34         2.152.320 Microsoft.VisualStudio.Shell.15.0.dll
02/11/2021  06:34           336.248 Microsoft.VisualStudio.Shell.Framework.dll
02/11/2021  06:34           907.640 Microsoft.VisualStudio.Telemetry.dll
02/11/2021  06:34            18.944 Microsoft.VisualStudio.TestPlatform.Extension.Xunit.Xml.TestAdapter.dll
02/11/2021  06:34            18.944 Microsoft.VisualStudio.TestPlatform.Extension.Xunit.Xml.TestLogger.dll
02/11/2021  06:34           295.272 Microsoft.VisualStudio.Text.Data.dll
02/11/2021  06:34           227.176 Microsoft.VisualStudio.Text.Internal.dll
02/11/2021  06:34           155.496 Microsoft.VisualStudio.Text.Logic.dll
02/11/2021  06:34           275.816 Microsoft.VisualStudio.Text.UI.dll
02/11/2021  06:34           136.040 Microsoft.VisualStudio.Text.UI.Wpf.dll
02/11/2021  06:34           523.640 Microsoft.VisualStudio.Threading.dll
02/11/2021  06:34           562.568 Microsoft.VisualStudio.Utilities.dll
02/11/2021  06:34            35.176 Microsoft.VisualStudio.Utilities.Internal.dll
02/11/2021  06:34            32.624 Microsoft.VisualStudio.Validation.dll
02/11/2021  06:34            26.496 Microsoft.Win32.Registry.dll
02/11/2021  06:34           198.144 Moq.dll
02/11/2021  06:34           192.512 Nerdbank.Streams.dll
02/11/2021  06:34           701.992 Newtonsoft.Json.dll
02/11/2021  06:34            97.656 NuGet.Common.dll
02/11/2021  06:34           128.616 NuGet.Configuration.dll
02/11/2021  06:34           104.016 NuGet.Frameworks.dll
02/11/2021  06:34           624.520 NuGet.Packaging.dll
02/11/2021  06:34           721.488 NuGet.Protocol.dll
02/11/2021  06:34            63.080 NuGet.Resolver.dll
02/11/2021  06:34            46.976 NuGet.Versioning.dll
02/11/2021  06:47    <DIR>          pl
02/11/2021  06:47    <DIR>          pt
02/11/2021  06:47    <DIR>          pt-BR
02/11/2021  06:47    <DIR>          ref
02/11/2021  06:47           208.893 rehydrate.cmd
02/11/2021  06:47    <DIR>          ru
02/11/2021  06:47    <DIR>          runtimes
02/11/2021  06:34             6.144 SQLitePCLRaw.batteries_v2.dll
02/11/2021  06:34            46.592 SQLitePCLRaw.core.dll
02/11/2021  06:34             9.728 SQLitePCLRaw.nativelibrary.dll
02/11/2021  06:34            57.344 SQLitePCLRaw.provider.dynamic_cdecl.dll
02/11/2021  06:34           381.296 StreamJsonRpc.dll
02/11/2021  06:34            20.856 System.Buffers.dll
02/11/2021  06:34           189.312 System.Collections.Immutable.dll
02/11/2021  06:34            24.840 System.Composition.AttributedModel.dll
02/11/2021  06:34            59.128 System.Composition.Convention.dll
02/11/2021  06:34            62.184 System.Composition.Hosting.dll
02/11/2021  06:34            29.928 System.Composition.Runtime.dll
02/11/2021  06:34            64.760 System.Composition.TypedParts.dll
02/11/2021  06:34            98.184 System.Diagnostics.DiagnosticSource.dll
02/11/2021  06:34            71.552 System.IO.Pipelines.dll
02/11/2021  06:34           141.184 System.Memory.dll
02/11/2021  06:34           115.856 System.Numerics.Vectors.dll
02/11/2021  06:34           462.728 System.Reflection.Metadata.dll
02/11/2021  06:34            29.440 System.Reflection.TypeExtensions.dll
02/11/2021  06:34            16.768 System.Runtime.CompilerServices.Unsafe.dll
02/11/2021  06:34            33.672 System.Security.AccessControl.dll
02/11/2021  06:34            18.312 System.Security.Principal.Windows.dll
02/11/2021  06:34           761.392 System.Text.Encoding.CodePages.dll
02/11/2021  06:34            57.720 System.Text.Encodings.Web.dll
02/11/2021  06:34           292.728 System.Text.Json.dll
02/11/2021  06:34            27.528 System.Threading.AccessControl.dll
02/11/2021  06:34           174.472 System.Threading.Tasks.Dataflow.dll
02/11/2021  06:34            25.984 System.Threading.Tasks.Extensions.dll
02/11/2021  06:34            25.232 System.ValueTuple.dll
02/11/2021  06:47    <DIR>          tr
02/11/2021  06:34            21.928 xunit.abstractions.dll
02/11/2021  06:34            72.616 xunit.assert.dll
02/11/2021  06:34            22.016 Xunit.Combinatorial.dll
02/11/2021  06:34            55.208 xunit.core.dll
02/11/2021  06:34           250.792 xunit.execution.desktop.dll
02/11/2021  06:47                99 xunit.runner.json
02/11/2021  06:34            71.592 xunit.runner.reporters.net452.dll
02/11/2021  06:34           248.232 xunit.runner.utility.net452.dll
02/11/2021  06:34           116.136 xunit.runner.visualstudio.testadapter.dll
02/11/2021  06:47    <DIR>          zh-Hans
02/11/2021  06:47    <DIR>          zh-Hant
             162 archivos    116.123.792 bytes
              19 dirs  119.865.856.000 bytes libres

C:\h\w\A62208C0\w\B8B609E9\e>set DOTNET_ROLL_FORWARD=LatestMajor 

C:\h\w\A62208C0\w\B8B609E9\e>set DOTNET_ROLL_FORWARD_TO_PRERELEASE=1 

C:\h\w\A62208C0\w\B8B609E9\e>dotnet --info 
SDK de .NET (que refleje cualquier global.json):
 Version:   6.0.100-rc.2.21505.57
 Commit:    ab39070116

Entorno de tiempo de ejecuci�n:
 OS Name:     Windows
 OS Version:  10.0.18362
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\h\w\A62208C0\p\dotnet-cli\sdk\6.0.100-rc.2.21505.57\

Host (useful for support):
  Version: 6.0.0-rc.2.21480.5
  Commit:  6b11d64e7e

.NET SDKs installed:
  6.0.100-rc.2.21505.57 [C:\h\w\A62208C0\p\dotnet-cli\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 6.0.0-rc.2.21480.10 [C:\h\w\A62208C0\p\dotnet-cli\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 6.0.0-rc.2.21480.5 [C:\h\w\A62208C0\p\dotnet-cli\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 6.0.0-rc.2.21501.6 [C:\h\w\A62208C0\p\dotnet-cli\shared\Microsoft.WindowsDesktop.App]

To install additional .NET runtimes or SDKs:
  https://aka.ms/dotnet-download

C:\h\w\A62208C0\w\B8B609E9\e>dotnet test "Microsoft.CodeAnalysis.LanguageServer.Protocol.UnitTests.dll" --framework net472 --logger "xunit;LogFilePath=.\Microsoft.CodeAnalysis.LanguageServer.Protocol.UnitTests.dll_net472_x86_test_results.xml" 

Esto es .NET 6.0.
---------------------
Versi�n del SDK: 6.0.100-rc.2.21505.57

----------------
Se instal� un certificado de desarrollo con HTTPS para ASP.NET Core.
Para confiar en el certificado, ejecute "dotnet dev-certs https --trust" (solo Windows y macOS).
Obtenga m�s informaci�n sobre HTTPS: https://aka.ms/dotnet-https
----------------
Escriba su primera aplicaci�n: https://aka.ms/dotnet-hello-world
Descubra las novedades: https://aka.ms/dotnet-whats-new
Explore la documentaci�n: https://aka.ms/dotnet-docs
Notifique los problemas y busque el c�digo fuente en GitHub: https://github.com/dotnet/core
Use "dotnet --help" para ver los comandos disponibles o visite: https://aka.ms/dotnet-cli
--------------------------------------------------------------------------------------
Herramienta de l�nea de comandos de ejecuci�n de pruebas de Microsoft(R), versi�n 17.0.0
Copyright (c) Microsoft Corporation. Todos los derechos reservados.

Iniciando la ejecuci�n de pruebas, espere...
1 archivos de prueba en total coincidieron con el patr�n especificado.
[xUnit.net 00:00:16.03]     Microsoft.CodeAnalysis.LanguageServer.UnitTests.Miscellaneous.LspMiscellaneousFilesWorkspaceTests.TestLooseFile_MovedToRegisteredWorkspace [FAIL]
  Con error Microsoft.CodeAnalysis.LanguageServer.UnitTests.Miscellaneous.LspMiscellaneousFilesWorkspaceTests.TestLooseFile_MovedToRegisteredWorkspace [314 ms]
  Mensaje de error:
   Assert.Null() Failure
Expected: (null)
Actual:   Document { FilePath = "C:/SomeFile.cs", Folders = [], Id = (DocumentId, #53fc56a0-a414-4480-a5eb-2e10c30f0e99 - C:/SomeFile.cs), Name = "C:/SomeFile.cs", Project = Project { AdditionalDocumentIds = [...], AdditionalDocuments = [...], AllProjectReferences = [...], AnalyzerConfigDocuments = [...], AnalyzerOptions = AnalyzerOptions { ... }, ... }, ... }
  Seguimiento de la pila:
     en Xunit.Assert.Null(Object object) en C:\projects\xunit\src\xunit.assert\Asserts\NullAsserts.cs:l�nea 31
   en Microsoft.CodeAnalysis.LanguageServer.UnitTests.Miscellaneous.LspMiscellaneousFilesWorkspaceTests.<TestLooseFile_MovedToRegisteredWorkspace>d__4.MoveNext() en /_/src/Features/LanguageServer/ProtocolUnitTests/Miscellaneous/LspMiscellaneousFilesWorkspaceTests.cs:l�nea 169
--- Fin del seguimiento de la pila de la ubicaci�n anterior donde se produjo la excepci�n ---
   en System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en Xunit.Sdk.TestInvoker`1.<>c__DisplayClass48_1.<<InvokeTestMethodAsync>b__1>d.MoveNext() en C:\projects\xunit\src\xunit.execution\Sdk\Frameworks\Runners\TestInvoker.cs:l�nea 264
--- Fin del seguimiento de la pila de la ubicaci�n anterior donde se produjo la excepci�n ---
   en System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en Xunit.Sdk.ExecutionTimer.<AggregateAsync>d__4.MoveNext() en C:\projects\xunit\src\xunit.execution\Sdk\Frameworks\ExecutionTimer.cs:l�nea 48
--- Fin del seguimiento de la pila de la ubicaci�n anterior donde se produjo la excepci�n ---
   en System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en Xunit.Sdk.ExceptionAggregator.<RunAsync>d__9.MoveNext() en C:\projects\xunit\src\xunit.core\Sdk\ExceptionAggregator.cs:l�nea 90
[xUnit.net 00:00:22.51]     Microsoft.CodeAnalysis.LanguageServer.UnitTests.Symbols.DocumentSymbolsTests.TestGetDocumentSymbolsAsync__WithLocals [SKIP]
  Omitidas Microsoft.CodeAnalysis.LanguageServer.UnitTests.Symbols.DocumentSymbolsTests.TestGetDocumentSymbolsAsync__WithLocals [1 ms]
[xUnit.net 00:00:30.98]     Microsoft.CodeAnalysis.LanguageServer.UnitTests.FoldingRanges.FoldingRangesTests.TestGetFoldingRangeAsync_Comments [SKIP]
  Omitidas Microsoft.CodeAnalysis.LanguageServer.UnitTests.FoldingRanges.FoldingRangesTests.TestGetFoldingRangeAsync_Comments [1 ms]
[xUnit.net 00:00:31.03]     Microsoft.CodeAnalysis.LanguageServer.UnitTests.FoldingRanges.FoldingRangesTests.TestGetFoldingRangeAsync_Regions [SKIP]
  Omitidas Microsoft.CodeAnalysis.LanguageServer.UnitTests.FoldingRanges.FoldingRangesTests.TestGetFoldingRangeAsync_Regions [1 ms]
[xUnit.net 00:00:31.49]     Microsoft.CodeAnalysis.LanguageServer.UnitTests.Symbols.WorkspaceSymbolsTests.TestGetWorkspaceSymbolsAsync_Local [SKIP]
  Omitidas Microsoft.CodeAnalysis.LanguageServer.UnitTests.Symbols.WorkspaceSymbolsTests.TestGetWorkspaceSymbolsAsync_Local [1 ms]
Results File: C:\h\w\A62208C0\w\B8B609E9\e\Microsoft.CodeAnalysis.LanguageServer.Protocol.UnitTests.dll_net472_x86_test_results.xml

Con error! - Con error:     1, Superado:   196, Omitido:     4, Total:   201, Duraci�n: 31 s - Microsoft.CodeAnalysis.LanguageServer.Protocol.UnitTests.dll (net472)

C:\h\w\A62208C0\w\B8B609E9\e>set _commandExitCode=1 

C:\h\w\A62208C0\w\B8B609E9\e>C:\python3.9.1\python.exe C:\h\w\A62208C0\p\reporter\run.py https://dev.azure.com/dnceng/ public 41743470 eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsIng1dCI6Im9PdmN6NU1fN3AtSGpJS2xGWHo5M3VfVjBabyJ9.eyJuYW1laWQiOiJlNDI4NWM4Yy0zZmQ1LTQyNjctOWIxYy00MjE5NWM0N2E1NTMiLCJzY3AiOiJhcHBfdG9rZW4iLCJhdWkiOiI5MGQ4MDFmNy0zOTk5LTQ5OWItYjQ4Zi1lMTNkNGMwMzkwOTkiLCJzaWQiOiIxNGQ5MzYxMS1jZjBjLTQ3OGEtOTk4Ny01MWU5ZjQxNTdjYmYiLCJCdWlsZElkIjoiOWVlNmQ0NzgtZDI4OC00N2Y3LWFhY2MtZjZlNmQwODJhZTZkOzE0NTAwNTgiLCJwcGlkIjoidnN0ZnM6Ly8vQnVpbGQvQnVpbGQvMTQ1MDA1OCIsIm9yY2hpZCI6Ijk1NTdhOTFjLTQ1Y2EtNDU2OS1iZmM1LThhMTQ2OGUxZThhZC50ZXN0X3dpbmRvd3NfZGVza3RvcF9zcGFuaXNoX3JlbGVhc2VfMzIuX19kZWZhdWx0IiwiaXNzIjoiYXBwLnZzdG9rZW4udmlzdWFsc3R1ZGlvLmNvbSIsImF1ZCI6ImFwcC52c3Rva2VuLnZpc3VhbHN0dWRpby5jb218dnNvOmI1NWRlNGVkLTRiNWEtNDIxNS1hOGU0LTBhMGE1ZjcxZTdkOCIsIm5iZiI6MTYzNTgzMzY2NSwiZXhwIjoxNjM1ODQyMDY1fQ.Mf4jHD6TprYfDksx5lotIqWw0fSU_bzFrA0-3Tifz3dP8rXb-CIPSlaK4ZOpkFsaqtkNedVjd_RipPlM2Fow2mgi5np8arHATOk0-jqySgRJtwa-HvIYfaUrYxl1GIhMrtRU6Yrbzo9iIy2swBa9g6gPtuThyPBTc-5n0GW4Z_IBNGLN-H93I9FysGHNHn4JWFPm0l0LACBeJzzgfQw5SOkjEbWIwQ_mtR8IPojqqnKHSlMPGznXidoiMqXxerppgzfSt0Us5LVwwHkeJu2ZDE79V2TvS29nWcnwb-_gxlVn529s-p305xfdZIAXi2dvMm1n-Q94nXnt0J-5wWHGpw   || exit /b 
2021-11-02 06:48:10,049: INFO: 3204: run(48): main: Beginning reading of test results.
2021-11-02 06:48:10,049: INFO: 3204: __init__(48): get_log_files: Searching 'C:\h\w\A62208C0\w\B8B609E9\e\..' for log files
2021-11-02 06:48:10,049: INFO: 3204: __init__(66): construct_log_list: Generated log list: console.a12e777b.log:
  https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-roslyn-refs-heads-release-dev171-vs-de4d2ed69194134b859b/Microsoft.CodeAnalysis.LanguageServer.Protocol.UnitTests.dll/console.a12e777b.log?sv=2019-07-07&se=2021-11-22T06%3A27%3A01Z&sr=c&sp=rl&sig=TsBVMtlk%2FuSphEQwsSzLnF9pUu%2F6gwjM3wlDsyuj27A%3D

2021-11-02 06:48:10,049: INFO: 3204: __init__(90): read_results: Searching 'C:\h\w\A62208C0\w\B8B609E9\e' for test results files
2021-11-02 06:48:10,049: INFO: 3204: __init__(96): read_results: Found results file C:\h\w\A62208C0\w\B8B609E9\e\Microsoft.CodeAnalysis.LanguageServer.Protocol.UnitTests.dll_net472_x86_test_results.xml with format xunit
2021-11-02 06:48:10,049: INFO: 3204: __init__(90): read_results: Searching 'C:\h\w\A62208C0\w\B8B609E9\uploads' for test results files
2021-11-02 06:48:10,049: INFO: 3204: packing_test_reporter(30): report_results: Packing 201 test reports to 'C:\h\w\A62208C0\w\B8B609E9\e\__test_report.json'
2021-11-02 06:48:10,049: INFO: 3204: packing_test_reporter(33): report_results: Packed 66161 bytes
Found log 'C:\h\w\A62208C0\w\B8B609E9\e\..\console.a12e777b.log'
Uri 'https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-roslyn-refs-heads-release-dev171-vs-de4d2ed69194134b859b/Microsoft.CodeAnalysis.LanguageServer.Protocol.UnitTests.dll/console.a12e777b.log?sv=2019-07-07&se=2021-11-22T06%3A27%3A01Z&sr=c&sp=rl&sig=TsBVMtlk%2FuSphEQwsSzLnF9pUu%2F6gwjM3wlDsyuj27A%3D'

C:\h\w\A62208C0\w\B8B609E9\e>EXIT /b 1 
```

|Build|Kind|Start Time|
|---|---|---|
[1450058](https://dev.azure.com/dnceng/public/_build/results?buildId=1450058)|Rolling|2021-02-11|

**Comment by dibarbet** [comment]
Was fixed in #57444 but the branch doesn't have the change since it was merged after the snap

**Comment by vatsalyaagrawal** [comment]
Duplicate of #57422
